v22.1/show-jobs.md

```diff
@@ -119,12 +119,21 @@ You can filter jobs by using `SHOW JOBS` as the data source for a [`SELECT`](sel
 ~~~
 
 ~~~
-    job_id           |       job_type      |                    description                      |...
-+--------------------+---------------------+-----------------------------------------------------+...
-  438235476849557505 | AUTO CREATE STATS   | Table statistics refresh for defaultdb.public.users |...
-(1 row)
+    job_id           |       job_type                  |                    description                       |...
++--------------------+---------------------------------+------------------------------------------------------+...
+  786475982730133505 | AUTO SPAN CONFIG RECONCILIATION | reconciling span configurations                      |...
+  786483120403382274 | AUTO SQL STATS COMPACTION       | automatic SQL Stats compaction                       |...
+  786476180299579393 | AUTO CREATE STATS               | Table statistics refresh for movr.public.promo_codes |...
+...
+(8 rows)
 ~~~
 
+The job types of automatic jobs are:
+
+- `AUTO SPAN CONFIG RECONCILIATION`: a continuously running job that ensures that all declared [zone configurations](configure-zone.html) (`ALTER … CONFIGURE ZONE …`) are applied. For example, when `num_replicas = 7` is set on a table, the reconciliation job listens in on those changes and then informs the underlying [storage layer](architecture/storage-layer.html) to maintain 7 replicas for the table.
```

Comment:
I think we normally capitalize after colons that introduce list items, including elsewhere on this page:

```suggestion
- `AUTO SPAN CONFIG RECONCILIATION`: A continuously running job that ensures that all declared [zone configurations](configure-zone.html) (`ALTER … CONFIGURE ZONE …`) are applied. For example, when `num_replicas = 7` is set on a table, the reconciliation job listens in on those changes and then informs the underlying [storage layer](architecture/storage-layer.html) to maintain 7 replicas for the table.
```